src/App.vue

```diff
@@ -2,10 +2,12 @@
   <div id="app">
     <div class="noteList">
       <div class="addBtm"><b>+ Add note</b></div>
-      <div class="notes"></div>
+      <div class="notesListTi" v-for="(note, index) in notes" :key="index" @click="tap(index)">
```

Comment:
It's better to use the id of notes, although in the current situation the order of the notes won't change so it's fine to use `index` from `v-for`.

```suggestion
      <div class="notesListTi" v-for="note in notes" :key="note.id" @click="tap(note.id)">
```